rasa/shared/core/domain.py

```diff
@@ -185,11 +185,15 @@ def from_yaml(cls, yaml: Text, original_filename: Text = "") -> "Domain":
             raise e
 
     @classmethod
-    def from_dict(cls, data: Dict) -> "Domain":
+    def from_dict(
+        cls, data: Dict, duplicates: Optional[Dict[Text, List[Text]]] = None
+    ) -> "Domain":
         """Deserializes and creates domain.
 
         Args:
             data: The serialized domain.
+            duplicates: The dictionary of duplicated intents, slots, forms, etc when the
```

Comment:
I'd prefer having a more explicit docstring defining the key-value pairs of `duplicates` dict, I didn't quite know what to expect until looking further in the code in `merge`